lib/src/turf_adapters.dart

```diff
@@ -52,4 +56,8 @@ final class LineString extends turf.LineString {
     final line = turf.LineString.fromJson(json);
     return LineString(bbox: line.bbox, coordinates: line.coordinates);
   }
+
+  LineString.fromPoints({turf.BBox? bbox, required List<Point> points}) {
+    turf.LineString.fromPoints(points: points, bbox: bbox);
+  }
 }
```

Comment:
```suggestion
  LineString.fromPoints({turf.BBox? bbox, required List<Point> points}) : super.fromPoints(bbox: bbox, points: points);
}
```

Reply:
Neat syntax, thanks for the suggestion 👍